t04_refactored.py

```diff
@@ -160,8 +164,69 @@ def team_10_adv():
 
 
 def team_11_adv():
-    pass
-    # TODO Add your code here
+    """https://docs.google.com/document/d/1W-UPVLjpRIZoil4edlCxl8Dkjj91P0ky9xb35G1L55g/edit?usp=sharing
+    Frederick Oguine
+    Yin Kyay Wai
+    :return: none
+    """
+    from time import sleep
```

Comment:
This import isn't needed because imports belong at the beginning of the code (and this one is already there).